libmozdata/vcs_map.py

```diff
@@ -0,0 +1,112 @@
+import argparse
+import os
+import sys
+import urllib.request
+
+import hglib
+import requests
+from tqdm import tqdm
```

Comment:
I'd prefer to not have it as a requirement but we can use it if the import doesn't fail.

Reply:
I've just removed it in ad3b4a362329e49467b82ddc500edbd14227aed8.